src/m365/spo/commands/page/page-section-add.ts

```diff
@@ -90,8 +155,8 @@ class SpoPageSectionAddCommand extends SpoCommand {
         }
 
         if (typeof args.options.zoneEmphasis !== 'undefined') {
-          if (!(args.options.zoneEmphasis in ZoneEmphasis)) {
-            return 'The value of parameter zoneEmphasis must be None|Neutral|Soft|Strong';
+          if (SpoPageSectionAddCommand.ZoneEmphasis.indexOf(args.options.zoneEmphasis) === -1) {
+            return 'The value of parameter zoneEmphasis must be None|Neutral|Soft|Strong|Image|Gradient';
           }
         }
```

Comment:
A few comments here:
- Try to keep choices case-insensitive
- Try to avoid using `indexOf` as there are better alternatives.
- With the returning string, let's not hardcode the options but use our defined array

Reply:
Done